tools/corridor-geojson2csv.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+import csv
+import json
+
+
+def convert(file_path, idx_offset=1):
+    csv_path = file_path + ".csv"
```

Comment:
I think this will result in a file called `corridors.geojson.csv`, right?
Is that intended?